myrestapi/client/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Cristina Library</title>
+
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+</head>
+
+<body>
+    <div class="container">
+        <hr>
+        <h1>List of Cristina's books</h1>
+        <hr>
+
+        <div>
+            <div class="row" id="books">
+            </div>
+        </div>
+    </div>
+<!-- 
```

Comment:
Another nit/style point: Generally it's not great to check in comments out code. If the code was committed at one point you can always go back in the github history to see it.